### ISSUE TITLE
Cann't install php56-sphinx

### ISSUE THREAD

**Issue by Purelightme** [Homebrew/homebrew-php#4066]
## Description

My php is php56 installed through homebrew.Because my php is debug version,i use "brew install php56-sphinx -fs" to install php extensions suitable with my php.But i cannot install it successful,The output just like this:

## Context and Logs

$ brew install php56-sphinx -fs
Updating Homebrew...
==> Installing php56-sphinx from josegonzalez/php
==> Downloading https://pecl.php.net/get/sphinx-1.3.2.tgz
Already downloaded: /Users/purelightme/Library/Caches/Homebrew/php56-sphinx-1.3.2.tgz
==> /usr/local/opt/php56/bin/phpize
Last 15 lines from /Users/purelightme/Library/Logs/Homebrew/php56-sphinx/01.phpize:
2017-03-26 00:14:33 +0800

/usr/local/opt/php56/bin/phpize

Configuring for:
PHP Api Version:         20160303
Zend Module Api No:      20160303
Zend Extension Api No:   320160303
/Library/Developer/CommandLineTools/usr/bin/m4:aclocal.m4:3117: cannot open `build/ax_check_compile_flag.m4': No such file or directory
autom4te: /usr/bin/m4 failed with exit status: 1

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/josegonzalez/homebrew-php/issues

/usr/local/Homebrew/Library/Homebrew/utils/github.rb:226:in `raise_api_error': curl failed!  (GitHub::Error)
curl: (22) The requested URL returned error: 422 Unprocessable Entity
curl: (3) <url> malformed
	from /usr/local/Homebrew/Library/Homebrew/utils/github.rb:184:in `open'
	from /usr/local/Homebrew/Library/Homebrew/utils/github.rb:233:in `issues_matching'
	from /usr/local/Homebrew/Library/Homebrew/utils/github.rb:266:in `issues_for_formula'
	from /usr/local/Homebrew/Library/Homebrew/exceptions.rb:338:in `fetch_issues'
	from /usr/local/Homebrew/Library/Homebrew/exceptions.rb:334:in `issues'
	from /usr/local/Homebrew/Library/Homebrew/exceptions.rb:396:in `dump'
	from /usr/local/Homebrew/Library/Homebrew/brew.rb:130:in `rescue in <main>'
	from /usr/local/Homebrew/Library/Homebrew/brew.rb:31:in `<main>'

Hoping to be resolved.

**Comment by javian** [comment]
This looks like a connection error to me. Did you try it again ?

**Comment by Purelightme** [comment]
I got the same result after five attemps.It seesms not to be network'matter.Can you try it on your computer?

**Comment by javian** [comment]
@Purelightme I had no problems installing it here. What does you `brew doctor` output ?

**Comment by Purelightme** [comment]
$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry and just ignore them. Thanks!

Warning: Homebrew's sbin was not found in your PATH but you have installed
formulae that put executables in /usr/local/sbin.
Consider setting the PATH for example like so
  echo 'export PATH="/usr/local/sbin:$PATH"' >> ~/.zshrc

**Comment by Purelightme** [comment]
$ brew doctor
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry and just ignore them. Thanks!

Warning: Homebrew's sbin was not found in your PATH but you have installed
formulae that put executables in /usr/local/sbin.
Consider setting the PATH for example like so
  echo 'export PATH="/usr/local/sbin:$PATH"' >> ~/.zshrc

# purelightme @ purelightmedeMacBook-Pro in ~/django/myblog [21:06:26] C:1
$ source ~/.zshrc

# purelightme @ purelightmedeMacBook-Pro in ~/django/myblog [21:07:14]
$ brew doctor
Your system is ready to brew.

# purelightme @ purelightmedeMacBook-Pro in ~/django/myblog [21:07:21]
$ brew install php56-sphinx -fs
Updating Homebrew...
==> Installing php56-sphinx from josegonzalez/php
==> Downloading https://pecl.php.net/get/sphinx-1.3.2.tgz
Already downloaded: /Users/purelightme/Library/Caches/Homebrew/php56-sphinx-1.3.2.tgz
==> /usr/local/opt/php56/bin/phpize
Last 15 lines from /Users/purelightme/Library/Logs/Homebrew/php56-sphinx/01.phpize:
2017-03-26 21:08:17 +0800

/usr/local/opt/php56/bin/phpize

Configuring for:
PHP Api Version:         20160303
Zend Module Api No:      20160303
Zend Extension Api No:   320160303
/Library/Developer/CommandLineTools/usr/bin/m4:aclocal.m4:3117: cannot open `build/ax_check_compile_flag.m4': No such file or directory
autom4te: /usr/bin/m4 failed with exit status: 1

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/josegonzalez/homebrew-php/issues

/usr/local/Homebrew/Library/Homebrew/utils/github.rb:226:in `raise_api_error': curl failed!  (GitHub::Error)
curl: (6) Could not resolve host: api.github.com
curl: (3) <url> malformed
	from /usr/local/Homebrew/Library/Homebrew/utils/github.rb:184:in `open'
	from /usr/local/Homebrew/Library/Homebrew/utils/github.rb:233:in `issues_matching'
	from /usr/local/Homebrew/Library/Homebrew/utils/github.rb:266:in `issues_for_formula'
	from /usr/local/Homebrew/Library/Homebrew/exceptions.rb:338:in `fetch_issues'
	from /usr/local/Homebrew/Library/Homebrew/exceptions.rb:334:in `issues'
	from /usr/local/Homebrew/Library/Homebrew/exceptions.rb:396:in `dump'
	from /usr/local/Homebrew/Library/Homebrew/brew.rb:130:in `rescue in <main>'
	from /usr/local/Homebrew/Library/Homebrew/brew.rb:31:in `<main>'

**Comment by javian** [comment]
Can you show your `brew config` as well ?

**Comment by Merovex** [comment]
I'm running into the same problem.

```
configure: error: Cannot find libz

If reporting this issue please do so at (not Homebrew/brew or Homebrew/core):
https://github.com/josegonzalez/homebrew-php/issues

Error: curl failed! 
curl: (22) The requested URL returned error: 422 Unprocessable Entity
curl: (3) <url> malformed
```

## Brew Doctor

```                                                                   [ruby-2.3.1]
Please note that these warnings are just used to help the Homebrew maintainers
with debugging if you file an issue. If everything you use Homebrew for is
working fine: please don't worry and just ignore them. Thanks!

Warning: Some directories in your path end in a slash.
Directories in your path should not end in a slash. This can break other
doctor checks. The following directories should be edited:
  /sw/bin/
  /usr/local/
  /usr/local/bin/
  /Library/TeX/texbin/

Warning: Broken symlinks were found. Remove them with `brew prune`:
  /usr/local/bin/apm
  /usr/local/bin/atom
  /usr/local/lib/node_modules/firebase-tools/node_modules/.bin/base64url
  /usr/local/lib/node_modules/firebase-tools/node_modules/.bin/repeating
  /usr/local/share/ghostscript/9.07/Resource/Font/logo10.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/logo8.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/logo9.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/logobf10.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/logod10.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/logosl10.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/logosl8.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/logosl9.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/manfnt.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/wasy10.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/wasy5.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/wasy6.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/wasy7.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/wasy8.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/wasy9.pfb
  /usr/local/share/ghostscript/9.07/Resource/Font/wasyb10.pfb
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/HiraKakuPro-W3
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/HiraKakuPro-W6
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/HiraKakuProN-W3
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/HiraKakuProN-W6
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/HiraKakuStd-W8
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/HiraKakuStdN-W8
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/HiraMaruPro-W4
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/HiraMaruProN-W4
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/HiraMinPro-W3
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/HiraMinPro-W6
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/HiraMinProN-W3
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/HiraMinProN-W6
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/HiraginoSansGB-W3
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/HiraginoSansGB-W6
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/YuGo-Bold
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/YuGo-Medium
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/YuMin-Demibold
  /usr/local/share/ghostscript/9.16/Resource/CIDFont/YuMin-Medium
```

## Brew Config

```
HOMEBREW_VERSION: 1.1.11
ORIGIN: https://github.com/Homebrew/brew.git
HEAD: 394f9fa0aaa5854aa52bc589708a079665dcf462
Last commit: 3 weeks ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 484208f5422294641a1327580b54bad1425f25d5
Core tap last commit: 51 minutes ago
HOMEBREW_PREFIX: /usr/local
HOMEBREW_REPOSITORY: /usr/local/Homebrew
HOMEBREW_CELLAR: /usr/local/Cellar
HOMEBREW_BOTTLE_DOMAIN: https://homebrew.bintray.com
CPU: quad-core 64-bit haswell
Homebrew Ruby: 2.0.0-p648
Clang: 8.0 build 800
Git: 2.12.2 => /usr/local/bin/git
Perl: /usr/bin/perl
Python: /usr/bin/python
Ruby: /Users/bwilson/.rvm/rubies/ruby-2.3.1/bin/ruby
Java: 1.8.0_112
macOS: 10.12.3-x86_64
Xcode: 8.2.1
CLT: N/A
X11: N/A
```

**Comment by Purelightme** [comment]
why does phpize run error?
![2017-03-31 8 29 17](https://cloud.githubusercontent.com/assets/16385534/24550481/c9096550-1650-11e7-92d4-419615b3d98c.png)

**Comment by Merovex** [comment]
Any update on this? This break is preventing me from a full upgrade.

**Comment by javian** [comment]
@Merovex you are having a different issue so you should open your own and not use someone elses. You need to run `xcode-select --install` and the libz issue should be gone.

**Comment by javian** [comment]
@Purelightme can you try to reinstall/recompile php56 and see if that works ?

**Comment by angelxmoreno** [comment]
even when trying to reinstalling php56 i was getting the nasty 
```sh
checking for ZLIB support... yes
checking if the location of ZLIB install directory is defined... no
configure: error: Cannot find libz
```

but I ran `xcode-select --install` and saw something about not having xcode-cli or something like that. Point is that the missing libz went away. I reinstalled php56 ( though I might not have had to ). 

Everything is as expected and i can install `php56-sphinx` its dependencies and module that depend on it.
